lib/authentification/authentification_base/lib/src/blocs/auth_state_changed.dart

```diff
@@ -8,6 +8,10 @@
 
 import 'package:authentification_base/src/models/auth_user.dart';
 import 'package:firebase_auth/firebase_auth.dart';
+import 'package:rxdart/rxdart.dart';
+
+final authUserSubject = BehaviorSubject<AuthUser?>.seeded(null);
+Stream<AuthUser?> get authUserStream => authUserSubject.stream;
```

Comment:
This wraps the firebase auth stream, correct? Could you add your explanation on why its necessary to wrap it here as well?